R/interpret.R

```diff
@@ -317,6 +323,42 @@ Interpreter <- R6::R6Class(
                          PDP.1D = PDP.1D.list,
                          PDP.2D = PDP.2D.list)
       self$ale.grid <- ale.grid
+
```

Comment:
Before we run code in a function, you should check that the given parameters are valid, users will often mess up parameters, especially when they are strings, so we should have an appropriate error message if they pass a string that we don't support. For example: 

`
if (! (pdp.rank %in% c('Variance', 'Inflection Point', 'Predicted Risk', 'Feature Predicted Risk'))) {
stop("Method giving for pdp.rank is not supported, this must be one of \'Variance\', \'Inflection Point\', \'Predicted Risk\', \'Feature Predicted Risk\' ")
}
`